### PR TITLE
Add Nhost

### PR DESCRIPTION
**Pull request by morganmspencer**
This adds Nhost to the providers by bringing #20 up to speed with most current updates, specifically the UI.

Closes #20

### CODE REVIEW

File: redwood.toml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 [web]
   port = 8910
   apiProxyPath = "/.netlify/functions"
-  includeEnvironmentVariables = ['AUTH0_DOMAIN', 'AUTH0_CLIENT_ID', 'AUTH0_REDIRECT_URI', 'AUTH0_AUDIENCE', 'AZURE_ACTIVE_DIRECTORY_CLIENT_ID', 'AZURE_ACTIVE_DIRECTORY_AUTHORITY', 'AZURE_ACTIVE_DIRECTORY_REDIRECT_URI', 'AZURE_ACTIVE_DIRECTORY_LOGOUT_REDIRECT_URI', 'MAGICLINK_PUBLIC', 'FIREBASE_API_KEY', 'FIREBASE_AUTH_DOMAIN', 'FIREBASE_DATABASE_URL', 'FIREBASE_PROJECT_ID', 'FIREBASE_STORAGE_BUCKET', 'FIREBASE_MESSAGING_SENDER_ID', 'FIREBASE_APP_ID', 'SUPABASE_KEY', 'SUPABASE_URL']
+  includeEnvironmentVariables = ['AUTH0_DOMAIN', 'AUTH0_CLIENT_ID', 'AUTH0_REDIRECT_URI', 'AUTH0_AUDIENCE', 'AZURE_ACTIVE_DIRECTORY_CLIENT_ID', 'AZURE_ACTIVE_DIRECTORY_AUTHORITY', 'AZURE_ACTIVE_DIRECTORY_REDIRECT_URI', 'AZURE_ACTIVE_DIRECTORY_LOGOUT_REDIRECT_URI', 'MAGICLINK_PUBLIC', 'FIREBASE_API_KEY', 'FIREBASE_AUTH_DOMAIN', 'FIREBASE_DATABASE_URL', 'FIREBASE_PROJECT_ID', 'FIREBASE_STORAGE_BUCKET', 'FIREBASE_MESSAGING_SENDER_ID', 'FIREBASE_APP_ID', 'SUPABASE_KEY', 'SUPABASE_URL', 'NHOST_BACKEND_URL']
```
Comment (Tobbe) [inline redwood.toml:4]: Ugh. Can we please make this multiline?

```
# Line breaks are OK when inside arrays
hosts = [
  "alpha",
  "omega"
]
```
https://github.com/toml-lang/toml/blob/master/README.md

**Comment by morganmspencer** [comment]
@Tobbe I agree about the array in the redwood.toml. It actually caused some troubles when trying to implement this PR. I've updated it to a multiline array

**Comment by Tobbe** [comment]
Maybe I'm missing something here, but shouldn't `decoded` be an actual decoded token?

image

**Comment by Tobbe** [comment]
This is supabase, there you get a decoded token

image

**Comment by Tobbe** [comment]
I went ahead and merged. The decoded issue has nothing to do with this PR.
@nunopato Do you have any ideas about why it's not decoding the token? (See my previous comment(s))

**Comment by nunopato** [comment]
@Tobbe will look into it, and report back later today.

**Comment by dthyresson** [comment]
@Tobbe and @nunopato The reason is that there is no decoder implemented in: https://github.com/redwoodjs/redwood/tree/main/packages/api/src/auth/decoders

If it is to be decoded with a secret, see: https://github.com/redwoodjs/redwood/blob/main/packages/api/src/auth/decoders/supabase.ts for an example.

**Comment by Tobbe** [comment]
@dthyresson Thanks DT. That jogged my memory. We did discuss this when the provider was added. 
https://github.com/redwoodjs/redwood/pull/1725#discussion_r574268636

**Comment by nunopato** [comment]
@Tobbe @dthyresson is this good to go or do you want me to add a local decoder?

**Comment by Tobbe** [comment]
@nunopato Please have a look here: https://github.com/redwoodjs/redwood/pull/2350

**Pull request merged**